Resources/Prototypes/_Shitmed/Entities/Structures/Machines/autodoc.yml

```diff
@@ -77,6 +77,8 @@
     speechSounds: Pai
   - type: Storage
     clickInsert: false # incredibly annoying
+    openOnActivate: false # DeltaV - also annoying
```

Comment:
nothing else happens when you press E remove this

Reply:
Eww it's because "Toggle UI" is alphabetically after "Open Storage" and "Close Storage".

Reply:
verb priority is the solution here yes